CMakeLists.txt

```diff
@@ -200,10 +206,13 @@ target_link_libraries(molecular-gfx PUBLIC
 	molecular::meshfile
 	Eigen3::Eigen
 	OpenGL::GL
-	glfw
 	Threads::Threads
 )
 
+if(GLFW_FOUND)
+	target_link_libraries(molecular-gfx PUBLIC glfw)
+endif(GLFW_FOUND)
+
 add_subdirectory(examples)
```

Comment:
`add_subdirectory(examples)` should probably also be encompassed in the `if(GLFW_FOUND)`.